.github/workflows/build-and-test.yml

```diff
@@ -7,9 +7,33 @@ on:
     branches: [master, dev]
 
 jobs:
+  check-diff:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      frontend: ${{ steps.check-diff.outputs.frontend }}
+      cloud-functions: ${{ steps.check-diff.outputs.cloud-functions }}
+
```

Comment:
Let's keep stylistically consistent with the other jobs by removing these empty lines